enclave-tls/src/include/enclave-tls/api.h

```diff
@@ -68,18 +68,21 @@ typedef struct {
 	} quote_sgx_ecdsa;
 } enclave_tls_conf_t;
 
-#define ENCLAVE_TLS_API_VERSION_1                   1
-#define ENCLAVE_TLS_API_VERSION_MAX                 ENCLAVE_TLS_API_VERSION_1
-#define ENCLAVE_TLS_API_VERSION_DEFAULT             ENCLAVE_TLS_API_VERSION_1
+#define	ENCLAVE_TLS_API_VERSION_1			1
+#define	ENCLAVE_TLS_API_VERSION_MAX			ENCLAVE_TLS_API_VERSION_1
+#define	ENCLAVE_TLS_API_VERSION_DEFAULT			ENCLAVE_TLS_API_VERSION_1
 
-#define ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK_SHIFT    0
-#define ENCLAVE_TLS_CONF_FLAGS_PRIVATE_MASK_SHIFT   32
+#define	ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK_SHIFT	0
+#define	ENCLAVE_TLS_CONF_FLAGS_PRIVATE_MASK_SHIFT	32
+#define	ENCLAVE_TLS_CONF_FLAGS_PARAAUTO_MASK_SHIFT	33
 
-#define ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK          (ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK << ENCLAVE_TLS_CONF_FLAGS_PRIVATE_MASK_SHIFT
+#define	ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK		(ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK << ENCLAVE_TLS_CONF_FLAGS_PRIVATE_MASK_SHIFT
 
-#define ENCLAVE_TLS_CONF_FLAGS_MUTUAL               (1UL << ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK_SHIFT)
+#define	ENCLAVE_TLS_CONF_FLAGS_MUTUAL			(1UL << ENCLAVE_TLS_CONF_FLAGS_GLOBAL_MASK_SHIFT)
 
-#define ENCLAVE_TLS_CONF_FLAGS_SERVER               (1UL << ENCLAVE_TLS_CONF_FLAGS_PRIVATE_MASK_SHIFT)
+#define	ENCLAVE_TLS_CONF_FLAGS_SERVER			(1UL << ENCLAVE_TLS_CONF_FLAGS_PRIVATE_MASK_SHIFT)
+
+#define	ENCLAVE_TLS_CONF_VERPARA_AUTO			(1UL << ENCLAVE_TLS_CONF_FLAGS_PARAAUTO_MASK_SHIFT)
```

Comment:
What is happening here???

Reply:
table is not consistent in every development enviroment.

Reply:
I will introduce clang-format to handle this issue, like what `go fmt` does.

Reply:
ok. thanks